tests/klay_test_blockchain_test.go

```diff
@@ -65,6 +65,7 @@ type BCData struct {
 	engine             consensus.Istanbul
 	genesis            *blockchain.Genesis
 	governance         *governance.Governance
+	rewardDistributor  *reward.RewardDistributor
```

Comment:
@laggu These additions are not removal of unused code. Does it mean that these test codes ran using old codes? I think it would be better if this change were included in previous PRs.

Reply:
@andybclee yes. it does. The test codes used old codes
I'll be more careful..